miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/BarcodableSearchRestController.java

```diff
@@ -0,0 +1,72 @@
+package uk.ac.bbsrc.tgac.miso.webapp.controller.rest;
+
+import static uk.ac.bbsrc.tgac.miso.core.util.LimsUtils.isStringEmptyOrNull;
+
+import java.io.IOException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.io.IOException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)